src/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import CountryPage from './Components/Country/CountryContainer';
+import Naving from './Components/Navbar';
+import Home from './Components/Home';
+import store from './redux/configureStore';
+import Region from './Components/Regions/RegionContainer';
+
+ReactDOM.render(
+  <Provider store={store}>
+    <React.StrictMode>
+      <Router>
+        <Naving />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/country" element={<CountryPage />} />
+          <Route path="/region" element={<Region />} />
```

Comment:
- [OPTIONAL] It will be good for each country's detail page to have its unique route within the SPA. e.g. If I click on the Tunisia country card the URL should be `https://covid-metrics-webapp.herokuapp.com/region/Tunisia`, I suggest using the React Router hook([useParams](https://reactrouter.com/docs/en/v6/api#useparams)).